test/homework_test/01_variables_test/01_variables_tests.cpp

```diff
@@ -12,3 +13,15 @@ TEST_CASE("Verify sum_numbers function")
 	REQUIRE(add_numbers(10, 10) == 20);
 }
 
+TEST_CASE("Test Get Sales Tax.")
+{
+	REQUIRE(get_sales_tax_amount(10) == 0.675);
+	REQUIRE(get_sales_tax_amount(20) == 1.35);
+
+}
+
+TEST_CASE("Test Get Tip Amount.")
+{
+	REQUIRE(get_tip_amount(20, 15) == 3);
+	REQUIRE(get_tip_amount(20, 20) == 4);
```

Comment:
Testing should be done with values  .15 and .2.
REQUIRE(get_tip_amount(20, .15) == 3);
REQUIRE(get_tip_amount(20, .20) == 4);